app/views/student/messages/index.html.haml

```diff
@@ -0,0 +1,14 @@
+.container.chat-container
+  %h1.py-4.text-dark オリアプ相談チャット
+  .h-100.my-3.border
+    .d-flex.flex-column.px-0
+      .list-group.w-100.px-1.py-1#messages
+        = render partial: 'shared/message', collection: @messages
+      .mt-auto.w-100.px-2.py-1
+        %form.mb-0
+          = hidden_field_tag '', current_student.id, id: 'user_id'
+          = hidden_field_tag '', current_student.id, id: 'chat_group_id'
+          .row.w-100.mx-0
+            .chat-box-wrapper.col-10
+              = text_area_tag '', '', id: 'chatInput', class: 'border chat-box'
+            = button_tag '送信', type: 'button', id: 'chatSubmitButton', class: 'btn btn-raised btn-info col-2 chat-submit-button'
```

Comment:
__[HAML-Lint]__

Line is too long. [128/80]


<!-- Comment by [SideCI](https://sideci.com) -->